### ISSUE TITLE
pipelinewise status gave (TypeError: an integer is required (got type bytes))

### ISSUE THREAD

**Issue by toileto** [jmriego/pipelinewise-target-bigquery#7]
Hi team,

Thanks for this great tool! I really want to try it out to extract data from MongoDB and export it to BigQuery. Unfortunately, after following the installation carefully, when I tried the `pipelinewise status` it gave me:

```
(pipelinewise) yusuf@local-ubuntu:~/Documents/Alodokter/pipelinewise$ pipelinewise status
Traceback (most recent call last):
  File "/home/yusuf/Documents/Alodokter/pipelinewise/.virtualenvs/pipelinewise/bin/pipelinewise", line 33, in <module>
    sys.exit(load_entry_point('pipelinewise', 'console_scripts', 'pipelinewise')())
  File "/home/yusuf/Documents/Alodokter/pipelinewise/.virtualenvs/pipelinewise/bin/pipelinewise", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/lib/python3.8/importlib/metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/yusuf/Documents/Alodokter/pipelinewise/pipelinewise/cli/__init__.py", line 12, in <module>
    from .pipelinewise import PipelineWise
  File "/home/yusuf/Documents/Alodokter/pipelinewise/pipelinewise/cli/pipelinewise.py", line 17, in <module>
    from joblib import Parallel, delayed, parallel_backend
  File "/home/yusuf/Documents/Alodokter/pipelinewise/.virtualenvs/pipelinewise/lib/python3.8/site-packages/joblib/__init__.py", line 119, in <module>
    from .parallel import Parallel
  File "/home/yusuf/Documents/Alodokter/pipelinewise/.virtualenvs/pipelinewise/lib/python3.8/site-packages/joblib/parallel.py", line 28, in <module>
    from ._parallel_backends import (FallbackToBackend, MultiprocessingBackend,
  File "/home/yusuf/Documents/Alodokter/pipelinewise/.virtualenvs/pipelinewise/lib/python3.8/site-packages/joblib/_parallel_backends.py", line 22, in <module>
    from .executor import get_memmapping_executor
  File "/home/yusuf/Documents/Alodokter/pipelinewise/.virtualenvs/pipelinewise/lib/python3.8/site-packages/joblib/executor.py", line 14, in <module>
    from .externals.loky.reusable_executor import get_reusable_executor
  File "/home/yusuf/Documents/Alodokter/pipelinewise/.virtualenvs/pipelinewise/lib/python3.8/site-packages/joblib/externals/loky/__init__.py", line 12, in <module>
    from .backend.reduction import set_loky_pickler
  File "/home/yusuf/Documents/Alodokter/pipelinewise/.virtualenvs/pipelinewise/lib/python3.8/site-packages/joblib/externals/loky/backend/reduction.py", line 125, in <module>
    from joblib.externals import cloudpickle  # noqa: F401
  File "/home/yusuf/Documents/Alodokter/pipelinewise/.virtualenvs/pipelinewise/lib/python3.8/site-packages/joblib/externals/cloudpickle/__init__.py", line 3, in <module>
    from .cloudpickle import *
  File "/home/yusuf/Documents/Alodokter/pipelinewise/.virtualenvs/pipelinewise/lib/python3.8/site-packages/joblib/externals/cloudpickle/cloudpickle.py", line 152, in <module>
    _cell_set_template_code = _make_cell_set_template_code()
  File "/home/yusuf/Documents/Alodokter/pipelinewise/.virtualenvs/pipelinewise/lib/python3.8/site-packages/joblib/externals/cloudpickle/cloudpickle.py", line 133, in _make_cell_set_template_code
    return types.CodeType(
TypeError: an integer is required (got type bytes)
```

Any idea why this happens?

Thanks for any help!

Warm regards,

**Comment by patrykkalinowski** [comment]
I have the same issue with tap-hubspot and tap-exchangeratesapi

**Comment by jmriego** [comment]
I might be wrong, but I think this is an issue with base PipelineWise and not specifically with the BigQuery target. This is the link to the main repo:
https://github.com/transferwise/pipelinewise

Not sure if related, but I had problems some time ago running with Python 3.8 and had to use 3.7. Just in case you want to try that

**Comment by eazhary** [comment]
HAd the same issue, downgraded to python 3.7 and everything worked fine. The issue is related to the version of joblib not compatible with python 3.8

**Comment by arjunbanker** [comment]
Also noticing this issue with Python 3.9

**Comment by toileto** [comment]
Guys, sorry I forgot to get back. Yes, the problem was solved by downgrading the Python to the 3.7 version as stated by @eazhary. Thank you for the discussion!
